Prog/TOPA/TOPA_desc_tables.sas

```diff
@@ -42,6 +42,46 @@ proc sort data=TOPA_unit_check;
   by Ward2022 id;
 run;
 
+** Comparing SSLs from realprop dataset unit counts in addresses dataset by notice IDs **;
+data TOPA_entire_prop; 
```

Comment:
This is a problematic merge because both data sets have multiple obs for each ID. The Topa_addresses data seems sufficient since it has the addresses and unit counts. If you want to show the property transactions, I think that is a separate Proc Print.